packages/nextcloud/lib/src/api/user_status.openapi.json

```diff
@@ -821,7 +821,10 @@
                                                         {
                                                             "$ref": "#/components/schemas/Private"
                                                         },
-                                                        {}
+                                                        {
+                                                            "type": "array",
+                                                            "maxLength": 0
+                                                        }
```

Comment:
This will probably need to be patched to an anyOf.
I think it was also broken before as `{}` just matches anything.